gytheio-content-handlers/gytheio-content-handler-webdav/src/main/java/org/gytheio/content/handler/webdav/WebDavContentReferenceHandlerImpl.java

```diff
@@ -189,6 +191,32 @@ public InputStream getInputStream(ContentReference contentReference, boolean wai
         }
     }
 
+    @Override
+    public long putFile(File sourceFile, ContentReference targetContentReference)
+            throws ContentIOException
+    {
+        try
+        {
+            FileInputStream fis = new FileInputStream(sourceFile);
```

Comment:
Shouldn't this first check `isContentReferenceSupported`?

Reply:
I left it out originally since it was done when calling putInputStream. However, I've added it anyway, to be explicit. Thanks for the feedback.